plugins/modules/create_kafka.py

```diff
@@ -75,13 +68,9 @@
   - name: Create kafka
     redhat.rhoask.create_kafka:
       name: "kafka_name"
-      instance_type: "x1"
-      billing_model: "standard"
+      plan: "developer.x1"
       cloud_provider: "aws"
       region: "us-east-1"
-      plan: "developer.x1"
-      billing_cloud_account_id: "123456789"
-      openshift_offline_token: "OPENSHIFT_CLUSTER_MANAGER_API_OFFLINE_TOKEN"
```

Comment:
It that would be our token to work then we picked the most confusing name. 
Even as insider I have no clue what this token is. I would call it refreshtoken or offline_token at most

Reply:
https://console.redhat.com/openshift/token/show# ...

Reply:
Long story short. We were supposed to get our own token page but this never happened. 
OpenShift Cluster manager is completely different product. We are using "their" token page to get refresh token as there is no other way. 

Proper names are:

- RefreshToken
- OfflineToken

Then doc should tell how to get it (and that will likely to change).

There is also other token page: https://console.redhat.com/ansible/automation-hub/token
Maybe we could use that token instead (it has ansible in name by coincidence)